src/views/ManageList.jsx

```diff
@@ -1,7 +1,65 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
 export function ManageList() {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+
+	const toastCSS = {
+		autoClose: 5000,
+		hideProgressBar: false,
+		closeOnClick: true,
+		pauseOnHover: true,
+		draggable: true,
+		progress: undefined,
+		theme: 'light',
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const res = await addItem(
+				localStorage.getItem('tcl-shopping-list-path'),
+				{ itemName, daysUntilNextPurchase },
+			);
+			toast.success('Item saved to database');
+		} catch (error) {
+			console.log(error);
+			toast.error('Item not saved');
+		}
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<>
+			<p>
+				Hello from the <code>/manage-list</code> page!
+			</p>
+			<form onSubmit={handleSubmit}>
+				<div>
+					<label htmlFor="itemName">Name of item: </label>
+					<input
+						value={itemName}
+						onChange={(e) => setItemName(e.target.value)}
+						type="text"
+						name="itemName"
+						id="itemName"
+						required
+					/>
+				</div>
+				<select
+					id="daysUntilNextPurchase"
+					value={daysUntilNextPurchase}
+					onChange={(e) => setDaysUntilNextPurchase(e.target.value)}
+				>
+					<option value={7}>Soon</option>
+					<option value={14}>Kind of soon</option>
+					<option value={30}>Not soon</option>
+				</select>
```

Comment:
Great use of the HTML select tag! But currently, we don't indicate what's the `select` for. It would be nice to add a label to it, just like you did for the name input.